src/session.cpp

```diff
@@ -225,12 +230,20 @@ class Impl : public dap::Session {
       errorLocked(format, vararg);
       va_end(vararg);
     }
-
+    int _vscprintf(const char* format, va_list pargs) {
```

Comment:
[Do not name functions with an initial underscore](https://stackoverflow.com/a/228797).

I'd recommend promoting the `va_copy` up to `errorLocked` and remove this function.